### DISCUSSION TITLE
What database permissions are required?

### DISCUSSION THREAD

**Discussion by cberes** [javers/javers#1319] [Q&A]
Hi there, I'm in the process of adding Javers to an app. When I tried to do my first commit, I encountered an error caused by our postgres user's missing permissions.

Rather than adding permissions one by one as we encounter errors, we'd like to know all the required database permissions for Javers. Is there a list of required permissions?

I don't need the permissions to create the tables/indexes/sequences, only the permissions for normal use after the database is already setup. Specifically I'm interested in the permissions for postgres.

**Comment by bartoszwalacik** [comment]
No special permissions are required for Javers, just the usual stuff so CREATE tables/indexes/sequences

**Reply by cberes** [reply]
Hi Bartosz, thank you for replying! I'm interested in the permissions needed _after_ we've created the tables/indexes/sequences.

My user has permissions for SELECT, INSERT, UPDATE, and DELETE. And I realized we need the permissions for the 3 sequences. Are there any other permissions a user would need?

**Reply by bartoszwalacik** [reply]
Hi @cberes , that should be all : SELECT, INSERT, UPDATE for tables and incrementing sequences
